lib/sacastats_web/controllers/character_controller.ex

```diff
@@ -20,10 +21,14 @@ defmodule SacaStatsWeb.CharacterController do
         |> redirect(to: Routes.character_path(conn, :character_search))
 
       {:ok, %PS2.API.QueryResult{data: body}} ->
+        next_rank = Map.get(body, "battle_rank")
+        {next_rank, ""} = Integer.parse(Map.get(next_rank, "value"))
+        next_rank = next_rank + 1
+
         character_stuff = %{
```

Comment:
```suggestion
        character = %{
```
Maybe we can rename `character_stuff` to just `character`.

Reply:
Good call.